### ISSUE TITLE
Request: ContextHub Extension Schema

### ISSUE THREAD

**Issue by tonypapousek** [Adobe-Marketing-Cloud/reactor-user-docs#50]
Would it be possible to get formatting details for the [AEM ContextHub](https://docs.adobelaunch.com/extension-reference/web/adobe-contexthub-extension#default-contexthub-data-layer) data layer type? The default schema as an example would also be helpful.

On a related note: 
I noticed launch-tutorials directory has contributing instructions. Do you accept pull requests for extension-reference, too? I've already signed the CLA for other Adobe repos.

**Comment by thebenrob** [comment]
Yes, we'll accept pull requests for anything you'd like to contribute.
Let me check with ContextHub extension developers for details.

**Comment by thebenrob** [comment]
From the AEM team: We don’t have any details around the schema documented except for its generation. The default JSON Schema was simply generated by iterating over all available stores on our We.Retail demo site. The extension already comes with a default schema which is shown when “Customer ContextHub Data Layer” is selected.
